setup.py

```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 try:
-    from setuptools import setup
+    from setuptools import setup, find_packages
 except ImportError:
```

Comment:
Yeah. Its not very often used anymore. I figure we can add it back when someone needs it.